os/logm/README.md

```diff
@@ -0,0 +1,52 @@
+# LogM
+logging module to route debug messages.  
+It queues the messages in buffer, and writes it out periodically into the stdout sequentially in logm task.
+
+## How to enable and configure
+Enable configuration of LogM through menuconfig.
+```
+cd $TIZENRT_BASEDIR
+cd os
+make menuconfig
+```
+Select menu.
+```
+Debug Options -> Logger Module -> Logger Module to y
+```
+Select options.  
+Both `printf to logm` and `syslog to logm` should be selected. If not, messages can NOT be shown sequentially.
```

Comment:
should? If you want to say "should", let's remove those configs and include them inside logm defaultly.